lib/rubygems/commands/sources_command.rb

```diff
@@ -34,6 +34,10 @@ def initialize
       options[:update] = value
     end
 
+    add_option '-f', '--[no-]force', "Automatically answer 'yes' to all confirmation prompts" do |value, options|
```

Comment:
Since the actually behaviour is that we skip showing the prompts at all, as opposed to showing them and answering `yest` automatically, we could maybe make the help text slightly more accurate by saying:

```suggestion
    add_option '-f', '--[no-]force', "Do show any confirmation prompts and behave as if 'yes' was always answered" do |value, options|
```

Reply:
Using that phrase slightly changed to say 'Do not show'

Reply:
Update pushed